app/src/matchmaking/match/match.cpp

```diff
@@ -52,7 +52,7 @@ void Match::addMoveData(const Player& player, int64_t measured_time_ms, bool leg
         ss << (move_data.score >= 0 ? '+' : '-');
         ss << std::fixed << std::setprecision(2) << (float(std::abs(move_data.score)) / 100);
     } else if (score_type == engine::ScoreType::MATE) {
-        uint64_t plies = board_.sideToMove() == Chess::Color::WHITE ? std::abs(move_data.score) * 2 
+        uint64_t plies = board_.sideToMove() == chess::Color::WHITE ? std::abs(move_data.score) * 2 
```

Comment:
`chess::Color::WHITE` doesn't this depend on which side is winning ?

Reply:
yeah, this one is outdated, this is the new one:
```
uint64_t plies = move_data.score > 0 ? std::abs(move_data.score) * 2 - 1
                                             : std::abs(move_data.score) * 2;
```